src/components/scene/vr-mode-ui.js

```diff
@@ -0,0 +1,188 @@
+var registerComponent = require('../../core/component').registerComponent;
```

Comment:
Devs will want to provide their own customized UIs so it's good to make this optional/configurable